fiatconnect-api.md

```diff
@@ -2267,6 +2268,28 @@ The `ifsc` or Indian Financial System Code is an 11-digit alpha-numeric code tha
 The `accountNumber` field is unique between individuals and no two banks or account holders can have the same account number.
 Banks use different starting codes for their branches for differentiation. (In India, bank account numbers usually contain 7 to 21 digits.)
 
+#### 8.3.2.6. `USDAccount`
```

Comment:
probly needs to be added to table of contents

Reply:
done.